api/db/migrations/20200327110821_save_acquired_partner_certification_key_on_certif_course_table.js

```diff
@@ -0,0 +1,23 @@
+const TABLE_NAME = 'certification-partner-acquisitions';
+const REF_CERTIF_COURSE_TABLE_NAME = 'certification-courses';
+const REF_BADGE_TABLE_NAME = 'badges';
+const COLUMN_KEY = 'partnerKey';
+
+exports.up = async (knex) => {
+
+  return knex.schema.createTable(TABLE_NAME, (table) => {
+    table
+      .integer('certificationCourseId')
+      .references(`${REF_CERTIF_COURSE_TABLE_NAME}.id`)
+      .notNullable()
+      .index();
+    table
+      .string(COLUMN_KEY)
+      .references(`${REF_BADGE_TABLE_NAME}.key`)
+      .notNullable();
```

Comment:
Ici peut-être qu'on pourrait mettre tous les noms en direct sans passer par des variables ? C'est pas ce qu'on fait d'habitude mais ça me paraît plus clair.

```suggestion
    table
      .integer('certificationCourseId')
      .references(`certification-courses.id`)
      .notNullable()
      .index();
    table
      .string('partnerKey')
      .references(`badges.key`)
      .notNullable();
```